docs/conf.py

```diff
@@ -21,11 +21,11 @@
 # -- Project information -----------------------------------------------------
 
 project = "ZGW Consumers"
-copyright = "2022, Maykin Media"
+copyright = "2023, Maykin Media"
```

Comment:
copyrights do not need to be changed, it's just the starting point.

Reply:
Looking online I see a lot of different answers, and some people also recommended not to include any date at all for websites. Anyway, I don't know much about the legal aspect of copyrights and I even wonder how it relates to the project's license